runtime/vm/LayoutFFITypeHelpers.cpp

```diff
@@ -204,10 +268,10 @@ void
 LayoutFFITypeHelpers::freeStructFFIType(ffi_type *ffiType)
 {
 	if ((NULL != ffiType)
-			&& ((FFI_TYPE_STRUCT == ffiType->type)
+		&& ((FFI_TYPE_STRUCT == ffiType->type)
 #if defined(J9ZOS390) && defined(J9VM_ENV_DATA64)
-				|| (FFI_TYPE_STRUCT_FF == ffiType->type)
-				|| (FFI_TYPE_STRUCT_DD == ffiType->type)
+		|| (FFI_TYPE_STRUCT_FF == ffiType->type)
+		|| (FFI_TYPE_STRUCT_DD == ffiType->type)
```

Comment:
At least one level of extra indentation should stay to clarify that it is part of the expression after `&&`.

Reply:
Fixed.